meshpy/utility.py

```diff
@@ -387,3 +387,63 @@ def get_env_variable(name, *, default="default_not_set"):
     elif default == "default_not_set":
         raise ValueError(f"Environment variable {name} is not set")
     return default
+
+
+def linear_time_transformation(
```

Comment:
Does this really belong into the general utility file? I think this is so closely related to the DBC monitor that we should add it as a free function there.

Reply:
I moved it to `function_utility`, I think this is a better place.